### PR TITLE
CHannels-806: Mobile Push Cleanup

### PR DESCRIPTION
**Pull request by alfrimpong**
[CHANNELS-806](https://segment.atlassian.net/browse/CHANNELS-806)

* Enforcing the deep link preset.
* Minor cleanup

**Note: This sendMobilePush action is not customer facing**

## Testing

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

- [x] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [x] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ ] [Segmenters] Tested in the staging environment

[CHANNELS-806]: https://segment.atlassian.net/browse/CHANNELS-806?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/destination-actions/src/destinations/engage/twilio/__tests__/send-mobile-push.test.ts [hunk 1]
```diff
@@ -84,94 +84,28 @@ const getDefaultExpectedNotifyApiReq = (extId: NonNullable<Payload['externalIds'
...
+          externalIds: [externalId]
+        }
+      })
       expect(responses[1].url).toStrictEqual(notifyReqUrl)
       expect(responses[1].status).toEqual(201)
       expect(responses[1].data).toMatchObject(externalId)
     })
 
-    it('should send notification to all subscribed devices', async () => {
-      const externalIds = [
```
Comment (alfrimpong) [inline packages/destination-actions/src/destinations/engage/twilio/__tests__/send-mobile-push.test.ts:101]: This situation where we get multiple push tokens in external ids for 1 event wont occur, hence why this test has been removed.

**Comment by joe-ayoub-segment** [comment]
Hi @alfrimpong , thanks for raising this PR. 
I'll schedule for review. 

Is it possible to split the changes for each Destination into separate PRs please? It's a policy we've have which makes it easier for changes to be rolled back (without affecting multiple Destinations).

**Comment by alfrimpong** [comment]
> Hi @alfrimpong , thanks for raising this PR. I'll schedule for review.
> 
> Is it possible to split the changes for each Destination into separate PRs please? It's a policy we've have which makes it easier for changes to be rolled back (without affecting multiple Destinations).

Hey Joe, there's only 1 destination and 1 action involved in this Pr.

**Review by joe-ayoub-segment — APPROVED** [review]

**Comment by joe-ayoub-segment** [comment]
Hi @alfrimpong , there was a flakey test from another Integration which needed to be fixed. 
Could you get the latest code from main so CI can pass for your PR please?

**Comment by alfrimpong** [comment]
> Hi @alfrimpong , there was a flakey test from another Integration which needed to be fixed. Could you get the latest code from main so CI can pass for your PR please?

Done

**Comment by alfrimpong** [comment]
> Hi @alfrimpong this PR has been deployed!

Muchisimo Gracias!!

**Pull request merged**